### ISSUE TITLE
pointer variables results in skipped function, without warning

### ISSUE THREAD

**Issue by MrYann** [jameskermode/f90wrap#43]
If a function uses pointer variables, the function is skipped, without warning.
Minimal example : 

module module_calcul
contains
subroutine evaluer_valeur(x)
real,dimension(:,:),pointer :: x
return
end subroutine evaluer_valeur
end module module_calcul

**Comment by jameskermode** [comment]
Yes, pointer variables are still not supported - pull requests welcome! A warning is printed if you run in verbose mode (``f90wrap -v``).

**Comment by jameskermode** [comment]
Comment from @MrYann from closed issue #47:
>Now let’s deal with the pointer arguments. Do you have any lead as to why they are not supported ? (i did no extensive testing, but f2py seems to support them)

I don't really remember, but I think I disabled pointer arguments because the format used for passing them varies from one compiler to another. I suggest temporarily turning off the removal of functions with pointer arguments and then we can see if they work, and if so with which compilers.
